### ISSUE TITLE
Xcode 7 Bitcode support for libidn

### ISSUE THREAD

**Issue by chrisballinger** [robbiehanson/XMPPFramework#562]
Needs to be recompiled with the `-fembed-bitcode` flag when building w/ Xcode 7.

**Comment by svtechnology** [comment]
Has anyone compiled libidn with bit code support?

**Comment by walsh2000** [comment]
+1
Has anyone compiled libidn with bitcode support?

**Comment by chrisballinger** [comment]
I am working on a podspec for libidn, which will solve the bitcode issue
for people using CocoaPods:
https://github.com/chrisballinger/libidn-podspec/blob/master/libidn.podspec

**Comment by walsh2000** [comment]
:muscle:  :+1:  That seems to work for me (It links, but I haven't tested it yet)

**Comment by galambalazs** [comment]
+1 this should be fixed in this project, seems easy enough

**Comment by peng90** [comment]
There's a [pull request](https://github.com/robbiehanson/XMPPFramework/pull/593) that fixes this issue. It works fine. Don't know why it's not merged yet. The CocoaPods approach didn't work for me. "idn-int.h" is empty.

**Comment by chrisballinger** [comment]
What happens if you have bitcode globally disabled for your app, but only
include a bitcode version of libidn here?

I'd prefer to get the libidn podspec working to avoid the issue entirely. I
was pretty close and believe it can be fixed.

**Comment by peng90** [comment]
@chrisballinger You're right that's a good point! For people who comes to this issue and just want a quick solution, I guess that pull request can serve as a temporary fix.... I look forward to your podspec solution 😄

**Comment by BesatZardosht** [comment]
@peng90 Thanks for your solution. I am new in this field! How should I compile it? How can I create linidn.a from build-libidn.sh?

**Comment by vikram17000** [comment]
Hi - wondering if this was ever moved forward? Thanks!
